src/org.opensuse.yast.ADSI.desktop.in

```diff
@@ -16,8 +16,9 @@ X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=adsi
 
 Icon=yast-adsi
-Exec=/usr/bin/xdg-su -c 'yast2 adsi qt'
+Exec=/usr/bin/xdg-su -c 'yast2 adsi'
```

Comment:
This is a correct change, but you check that it really works in ncurses? Just to be sure...

Reply:
Even if it doesn't, it has argument for ncurses published earlier in the file, so it will appear as an option in cli

Reply:
ok